extensions/panache/panache-common/runtime/src/main/java/io/quarkus/panache/common/Sort.java

```diff
@@ -218,24 +218,4 @@ public Sort and(String name, Direction direction) {
     public List<Column> getColumns() {
         return columns;
     }
-
-    /**
-     * Returns an SQL "order by" clause for the current sort columns and directions.
-     * 
-     * @return an SQL "order by" clause for the current sort columns and directions.
-     * @deprecated panache-common should be agnostic from the database implementation
-     */
-    @Deprecated
-    public String toOrderBy() {
-        StringBuilder sb = new StringBuilder(" ORDER BY ");
-        for (int i = 0; i < columns.size(); i++) {
-            Column column = columns.get(i);
-            if (i > 0)
-                sb.append(" , ");
-            sb.append(column.name);
-            if (column.direction != Direction.Ascending)
-                sb.append(" DESC");
-        }
-        return sb.toString();
-    }
```

Comment:
@FroMage @loicmathieu can you check that one?

Reply:
Yes, there is a replacement for it since a long time.
I keep it only because it was a public methods (but for internal usage) so it must be deleted at some times. And now is a good time for me :)